doc/ipython-notebooks/statistical_testing/mmd_two_sample_testing.ipynb

```diff
@@ -11,8 +11,8 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#### Heiko Strathmann - <a href=\"mailto:heiko.strathmann@gmail.com\">heiko.strathmann@gmail.com</a> - <a href=\"github.com/karlnapf\">github.com/karlnapf</a> - <a href=\"herrstrathmann.de\">herrstrathmann.de</a>\n",
-    "#### Soumyajit De - <a href=\"soumyajitde.cse@gmail.com \">soumyajitde.cse@gmail.com </a> - <a href=\"github.com/lambday\">github.com/lambday</a>"
+    "#### Heiko Strathmann - [heiko.strathmann@gmail.com](mailto:heiko.strathmann@gmail.com) - [github.com/karlnapf](http://github.com/karlnapf) - [herrstrathmann.de](http://herrstrathmann.de)\n",
```

Comment:
I think the notebooks render links as clickable when you just put the link itself.
For cases where the link text and the link are the same, you could try that. To avoid having the string there twice